tests/web_service/test_basic_request.py

```diff
@@ -0,0 +1,59 @@
+"""
+Test basic request by initializing test data.
+"""
+
+import logging
+import warnings
+
+from pydantic import ValidationError
+from pytest import mark, skip
+
+from poprox_concepts.api.recommendations import RecommendationRequest, RecommendationResponse
+from poprox_recommender.config import allow_data_test_failures
+from poprox_recommender.recommenders import recommendation_pipelines
+from poprox_recommender.request_generator import RequestGenerator
+from poprox_recommender.testing import auto_service as service
+from poprox_recommender.testing import mind_data
+
+logger = logging.getLogger(__name__)
+try:
+    PIPELINES = recommendation_pipelines().keys()
+except Exception as e:
+    warnings.warn("failed to load models, did you run `dvc pull`?")
+    if allow_data_test_failures():
+        skip("recommendation pipelines unavailable", allow_module_level=True)
+    else:
+        raise e
+
+
+@mark.docker
+@mark.parametrize("pipeline", PIPELINES)
+def test_basic_request(service, mind_data, pipeline):  # noqa: F811
+    """
+    Initialize request data
+    """
+    request_generator = RequestGenerator(mind_data())
```

Comment:
after the change to the fixture, this will become `mind_data` (without parentheses)

Reply:
This applies to other tests as well.